### ISSUE TITLE
Check 401 Error after Login with good credentials

### ISSUE THREAD

**Issue by BernardJeremy** [EIP-SAM/SAM-Solution-Node-js#61]
After a login with valid credentials, API still responds with 401 response

**Comment by Grenadingue** [comment]
With user authentication disabled in user API I'm then able to access the website from the webapp (`localhost:3000`). But we're not supposed to disable user authentication checks.

``` javascript
//
// Security check for each url of this kind : /api/public/*
//
module.exports.ensureLoggedOut = function (req, res, next) {
  next();
  // if (req.user) {
  //   logger.warn({ user: { id: req.user.id, name: req.user.name } }, 'Logged user is trying to access a public (logged-out) ressource');
  //   res.status(401).json({ error: 'Already logged-in' });
  // } else {
  //   next();
  // }
};

//
// Security check for each url of this kind : /api/logged-in/*
//
module.exports.ensureLoggedIn = function (req, res, next) {
  next();
  // if (req.user) {
  //   next();
  // } else {
  //   logger.warn('Non logged user is trying to access a protected ressource');
  //   res.status(401).json({ error: 'Not logged-in' });
  // }
};

//
// Security check for each url of this kind : /api/logged-in/admin/*
//
module.exports.ensureAdminLoggedIn = function (req, res, next) {
  next();
  // if (req.user.isAdmin) {
  //   next();
  // } else {
  //   logger.warn({ user: { id: req.user.id, name: req.user.name } }, 'Non admin user is trying to access a protected ressource');
  //   res.status(401).json({ error: 'Access denied' });
  // }
};
```

Cf: https://github.com/EIP-SAM/SAM-Solution-Server/blob/develop/managers/users.js#L105

Big note: When this [fix](https://github.com/EIP-SAM/SAM-Solution-Server/commit/a64f18a9bfb8315f8d1cc95c437a13205de10ce6) has been posted, the previous checks were disabled and mistakenly commited by a dev, that's why it worked before.

**Comment by Grenadingue** [comment]
Need to be done in the same time than #55

**Comment by michau-k** [comment]
Fixed in PR #99
